docs/intro/03-how-it-works.md

```diff
@@ -14,7 +14,28 @@ The server collects the results from all of the captured browsers and presents t
 
 This is only a very brief overview, as the internals of how Karma works aren't entirely necessary when using Karma.
 
-However, if you are interested in learning more, Karma itself originates from a university thesis, which goes into detail about the design
+## Outline of workflow.
+
+Here is roughly how Karma works:
+
+After starting up, Karma loads plugings and the configuration file, then starts its local web server which listened for connections.
+Any browser already watching the web address (localhost:9876 by default) will reconnect immediately. Karma also starts reporters
+which register for 'browser' events.
```

Comment:
What do you mean?

This sounds like it's saying that it starts those reporters immediately on server startup, whereas I assume you mean that it starts them on (re-)connections.

Reply:
Its part of loading the plugins; clarified.